xs2a-adapter-service-remote/src/main/java/de/adorsys/xs2a/adapter/remote/service/impl/RemoteAccountInformationService.java

```diff
@@ -56,13 +56,6 @@ public RemoteAccountInformationService(AccountInformationClient client) {
         objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
     }
 
-    public RemoteAccountInformationService(
```

Comment:
Why are you removing this constructor? This reverts a080d30

Reply:
not found usages of it, will revert back

Reply:
reverted back